### PR TITLE
feat: add Scheme filetypes / icons

### PR DESCRIPTION
**Pull request by nat-418**
This PR adds support for the [Scheme](https://www.scheme.org) programming language source filetypes.

### CODE REVIEW

File: src/output/icons.rs [hunk 1]
```diff
@@ -74,6 +74,7 @@ impl Icons {
     const LANG_RUBYRAILS: char  = '\u{e73b}';  // 
     const LANG_RUST: char       = '\u{e68b}';  // 
     const LANG_SASS: char       = '\u{e603}';  // 
+    const LANG_SCHEME: char     = '\u{e6b1}';  // 
```
Comment (PThorpe92) [inline src/output/icons.rs:77]: The `\u{e6b1}` I believe is correct but the comment you have 
image
seems to be the erlang logo

Comment (nat-418) [reply src/output/icons.rs:77]: Weird. This is what it looks like for me. I suppose this part of Unicode is the "Private Use Area" and a free-for-all. For what it is worth I am using the Nerd Fonts version of GitHub's Monaspace. I could change for `\u{03bb}`  (GREEK SMALL LETTER LAMDA) but I figured there was a reason the other icons were not in that range.
image

Comment (PThorpe92) [reply src/output/icons.rs:77]: That is odd because I actually use the same font :rofl:

**Review by MartinFillon — APPROVED** [review]
LGTM

**Pull request merged**